lib/train/transports/local.rb

```diff
@@ -22,18 +20,11 @@ class Connection < BaseConnection
       def initialize(options)
         super(options)
 
-        # While OS is being discovered, use the GenericRunner
-        @runner = GenericRunner.new
-        @runner.cmd_wrapper = CommandWrapper.load(self, options)
-
-        if os.windows?
-          # Attempt to use a named pipe but fallback to ShellOut if that fails
-          begin
-            @runner = WindowsPipeRunner.new
-          rescue PipeError
-            @runner = WindowsShellRunner.new
-          end
-        end
+        @runner = if options[:runner]
```

Comment:
I also wonder if this should be `:command_runner` and not just `:runner` in case we have file runners, etc. in the future.

Reply:
Don't see any reason not to do it. Will make it so.